platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/InputValidation.java

```diff
@@ -87,7 +95,7 @@ public static String sanitizeFilename(String filename) {
      * @param mimetype
      * @return true if the mime type is acceptable
      */
-    public static boolean checkForClientSideVulnerableMimeType(String mimetype) {
+    public static boolean isMimeTypeClientSideVulnerable(String mimetype) {
```

Comment:
❗️ This name change makes this method make no sense. The test asks: "is this vulnerable?" and if it is, you return false. If you want to change the name, either make it something like `isMimeTypeClientSideSafe` and keep its current semantics or switch the semantics and update all the calling locations.